cards/card.css

```diff
@@ -0,0 +1,82 @@
+@import url(/avatar/avatar.css);
+@import url(/buttons/buttons.css);
+
```

Comment:
The design of the card looks cool. You can try making them responsive for smaller screens by adding a few media queries so that there won't be any hassle later. Just a suggestion.